scripts/Makefile

```diff
@@ -90,8 +80,42 @@ ${GEOCINT_WORK_DIRECTORY}/geocint/installation/database_block_finished: ${GEOCIN
 	echo "Database initialisation step finished successfully"
 	touch $@
 
+# ------------------------ WEB Dashboard installation block ----------------------------
+${GEOCINT_WORK_DIRECTORY}/geocint/installation/nginx_installation: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/preparation_block_finished  ## install and configure nginx
+	sudo systemctl stop apache2
```

Comment:
How does apache appear on the system and why are we in the right to disable it silently?

Reply:
Apache is coming by default with Ubuntu 22.04 LTS.

I was going to make complete automatic install, including creation of nginx configurations. 
However, automatic nginx configuration turned out to be not feasible (for me),  so  currently it all is moved to manual steps and described in DOCUMENTATION.md